panel/util/__init__.py

```diff
@@ -124,8 +124,6 @@ def abbreviated_repr(value, max_length=25, natural_breaks=(',', ' ')):
         end_char = ''
         if isinstance(value, list):
             end_char = ']'
-        elif isinstance(value, OrderedDict):
```

Comment:
Is this OK? Or is special type cast or case needed for OrderedDict still here?

Reply:
I think it is fine to keep it.